checker/tests/tainting/ExtendHasQual.java

```diff
@@ -18,7 +18,7 @@ static class MyBuffer1 extends Buffer {}
   static class MyBuffer2 extends Buffer {}
 
   @HasQualifierParameter(Nullable.class)
-  // :: error: (missing.has.qual.param)
+  // :: error: (invalid.qual.param)
```

Comment:
A note on this change; the check for whether the argument to `@HasQualifierParameter` is a top type occurs before the check for whether a subclass also writes a `@HasQualifierParameter` annotation, that is why this test case changed.

I think I can also re-factor the code to make the check for top occur _after_, if this is the way it should be.